converse.js

```diff
@@ -3920,7 +3922,7 @@
                 if (chatbox.get('minimized')) {
                     chatbox.maximize();
                 } else {
-                    chatbox.trigger('show');
+                    chatbox.trigger('show', {focus: true});
                 }
```

Comment:
I'm unsure what I think of passing in an object like that. The only alternative I can think of right now, is to trigger a slightly different event, like `showAndFocus`. Not yet sure which is better.